<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMenuWindow</class>
 <widget class="QDialog" name="GameMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>114</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="WinLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string>Win</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LoseLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 4);</string>
         </property>
         <property name="text">
          <string>Lose</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="MoneyLabel">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="VerusButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Versus Bot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OnDeviceButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>On-Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OnlineButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Online</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>163</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>View Personal Data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>163</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="UsernameLabel">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="ScoreLabel">
         <property name="styleSheet">
          <string notr="true">padding: 5px;
color: rgb(85, 0, 127);
border-color:rgb(85, 0, 127);
border-style:solid;
border-width:2px;</string>
         </property>
         <property name="text">
          <string>Score</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LevelLabel">
         <property name="styleSheet">
          <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(0, 45, 134);
color: rgb(0, 45, 134);
border-radius: 100%;
padding: 5px;</string>
         </property>
         <property name="text">
          <string>Level</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
